locales/en_US.json

```diff
@@ -76,6 +76,14 @@
 			{
 				"question": "Will I need hacks to connect?",
 				"answer": "For the best experience on consoles, you will need to hack your system - specifially Aroma for Wii U and Luma3DS for 3DS. However, on Wii U, the hackless SSSL method is also available with limited functionality. See our <a href='/docs/install'>setup instructions</a> for details."
+			},
+			{
+				"question": "If I am banned on Nintendo Network, will I stay banned when using Pretendo?",
+				"answer": "We will not have access to Nintendo Network's bans, and all users will not be banned on our service. However, we will have rules to follow when using the service and failing to follow these rules could result in a ban."
+			},
+			{
+				"question": "Can I use cheats or mods online with Pretendo?",
+				"answer": "Only in private matches - cheating online with people who didn't consent (as in public matches) is a bannable offense. We regularly apply account and console bans to both Wii U and 3DS systems. Pretendo uses extra security measures that make traditional 'unban' methods like changing your serial number ineffective."
```

Comment:
Maybe this should be updated to note that "cheats" which do not give any advantage are also allowed, even in public matches? There have been questions about that. We don't have an "approved cheats list" or anything but common sense

Reply:
I think the public documentation should be as strict as possible, then we can choose to be lenient - common sense - as appropriate. (Discord takes the same approach to 3rd-party clients, for example.)

Outright saying "no advantage is okay" can lead to weird edge cases. In Splatoon, is forcing a specific stage an advantage? Is night mode an advantage? They're certainly very disruptive to the online experience and we don't want people doing it, but documenting carveouts like this could corner us.

Reply:
> They're certainly very disruptive to the online experience

Then this would obviously not be allowed. The official stance we've always had on cheats/mods in public sessions is that they are not allowed if they:

- Give you an advantage
- Disrupt the game for other users

Cheats like the ones you described would easily fall into the 2nd category.

The issue I see right now is that there will be a discrepancy between the information on the website and what we have officially said elsewhere (not just Discord, but on basically all of our socials as well), which is that cheats/mods which don't harm the experience of others are allowed in public sessions. This is different from how people like Discord handle this because we have explicitly *allowed* cheats/mods which don't fall into those categories. This can easily result in users not understanding what we do or do not actually allow, which can turn into more confusion, questions, and false reports.

Reply:
Honestly, with the way moderation has been going, I do wonder if changing our stance to be flat-out "no cheats"  - as per Discord - is a good option. Lots of people "accidentally" enable things or "forget it was on" or whatever. We could then have a small whitelist of definitely allowed things like Rival Unpatch for Splatoon or custom skins in Minecraft. That's a discussion to have in private though instead of on this FAQ update, so I'll adjust it as you suggested for the time being.